tests/cache/tests.py

```diff
@@ -1722,6 +1722,14 @@ def test_patch_cache_control(self):
                 parts = set(cc_delim_re.split(response['Cache-Control']))
                 self.assertEqual(parts, expected_cc)
 
+    def test_patch_cache_control_no_cache(self):
+        response = HttpResponse()
+        patch_cache_control(response, no_cache='SetCookie')
+        patch_cache_control(response, no_cache='foo')
+        self.assertEqual(response['Cache-Control'], 'no-cache=SetCookie, no-cache=foo')
+        patch_cache_control(response, no_cache=True)
+        self.assertEqual(response['Cache-Control'], 'no-cache')
```

Comment:
This work without this patch, i.e. `True` takes precedence over `no-cache` values without it. Extra tests are fine, but should be added in a separate commit, I would add new tests cases to `test_patch_cache_control()`:

- 1st commit with
```python
# no-cache,
('no-cache=SetCookie', {'no_cache': True}, {'no-cache'}),
('no-cache=SetCookie,no-cache=foo', {'no_cache': True}, {'no-cache'}),
```

- 2nd commit with a new feature and:
```python
('no-cache=SetCookie', {'no_cache': 'foo'}, {'no-cache=SetCookie', 'no-cache=foo'}),
```